packages/commonwealth/client/scripts/views/components/component_kit/cw_icons/cw_icon_lookup.ts

```diff
@@ -37,10 +37,13 @@ import * as Icons from './cw_icons';
 import { withPhosphorIcon } from './cw_phosphor_icons';
 
 export const iconLookup = {
+  arrowDownBlue500: Icons.CWArrowDownBlue500,
```

Comment:
I am not sure if using colors in the lookup is a good idea. Curious for your thoughts @Miaplacidus @masvelio

Reply:
Hey, @mzparacha!
I agree; I'd prefer to handle SVG colors using CSS. However, I believe I started doing this due to issues around updating the color/fill of certain SVGs via CSS. I'm not opposed to a conversation and/or second attempt at this (i.e. making a card to focus on consolidating SVGs and ensuring that they are colored as expected using CSS), but I think we should merge this work with the current icons.